server/fleet/activities.go

```diff
@@ -1694,6 +1694,7 @@ func (a ActivityAddedAppStoreApp) Documentation() (activity string, details stri
 - "software_title": Name of the App Store app.
 - "app_store_id": ID of the app on the Apple App Store.
 - "platform": Platform of the app (` + "`darwin`, `ios`, or `ipados`" + `).
+- "self_service": App installation can be initiated by dovice owner.
```

Comment:
```suggestion
- "self_service": App installation can be initiated by device owner.
```

Also, you should add the new field in the example JSON below.